src/System.IO/src/System/IO/MemoryStream.cs

```diff
@@ -474,41 +474,46 @@ public override int ReadByte()
             return _buffer[_position++];
         }
 
-
-        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+        public override void CopyTo(Stream destination, int bufferSize)
```

Comment:
In order to fix your build issues you are going to need a netcoreapp1.1 build configuration for the System.IO src project.